config/locales/en.yml

```diff
@@ -1212,6 +1217,7 @@ en:
   notice_to_many_principals_to_display: "There are too many results.\nNarrow down the search by typing in the name of the new member (or group)."
   notice_unable_delete_time_entry: "Unable to delete time log entry."
   notice_unable_delete_version: "Unable to delete version."
+  notice_value_information: "Your %{key_name} is: %{value}"
```

Comment:
I tried to use a more generalised name here on purpose since I think that this string can be re-used rather easily. However, If you think that translation strings should not be re-used at all then I'll go on an pick a name more suitable to the actual usage.